bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/preferences/EspresssifPreferencesPage.java

```diff
@@ -69,9 +72,44 @@ protected Control createContents(Composite parent)
 
 		addBuildSettings(mainComposite);
 
+		addToolsInstallationSettings(mainComposite);
+		
 		return mainComposite;
 	}
 
+	private void addToolsInstallationSettings(Composite mainComposite)
+	{
+		Group toolsInstallationGroup = new Group(mainComposite, SWT.SHADOW_ETCHED_IN);
+		toolsInstallationGroup.setText("Tools Installation");
+		toolsInstallationGroup.setLayout(new GridLayout(2, false));
+
+		Label githubAssetsLabel = new Label(toolsInstallationGroup, SWT.NONE);
+		githubAssetsLabel.setText("Mirror for GIT (IDF_GITHUB_ASSETS)");
+		gitAssetsText = new Text(toolsInstallationGroup, SWT.SINGLE | SWT.BORDER);
+		
+		Label pythonWheelsLabel = new Label(toolsInstallationGroup, SWT.NONE);
+		pythonWheelsLabel.setText("Mirror for python wheels (PIP_EXTRA_INDEX_URL)");
+		pythonWheelText = new Text(toolsInstallationGroup, SWT.SINGLE | SWT.NONE);
+		
+		GridData gitTextGridData = new GridData(SWT.FILL, SWT.CENTER, true, false);
```

Comment:
We could externalize these UI labels. Also, seems like the formatting is missing here.

Reply:
Done